apps/src/gamelab/animationLibrary.json

```diff
@@ -5184,7 +5184,7 @@
     "ore_iron": [
       "ore_iron"
     ],
-    "ore_ironAlt": [
+    "ore_ironalt": [
```

Comment:
Yeah, these are lifted from Kenney.  I imagine as we go through and curate we'll update the descriptive names to be more consistent, and the keys don't really matter as long as they match the filenames.